src/.nuget/NuGetRestore.ps1

```diff
@@ -1,5 +1,6 @@
 $NuGetExe = "$PSScriptRoot\NuGet.exe"
 
+& $NuGetExe install xunit.runner.console -version 2.0.0 -OutputDirectory "$PSScriptRoot\..\..\packages"
```

Comment:
@tmeschter, in dotnet/roslyn we ran into an issue, where running Unit Tests (which are currently built against 1.9.2) on any runner (console included) later than the 2.0.0 alpha runner we are currently on, results in no tests being discovered and no tests being run (which shows as 'passing' in the logs, but in actuality should be considered 'failed').

I imagine, unless you have retargeted the Unit Test assemblies themselves to v2.0 RTM or later, they are hitting the same issue and the Test Result XML logs are essentially empty.